rules/cloud/gcp/gcp_full_network_traffic_packet_capture.yml

```diff
@@ -0,0 +1,26 @@
+title: Google Full Network Traffic Packet Capture
+id: 980a7598-1e7f-4962-9372-2d754c930d0e
+description: Identifies potential full network packet capture in gcp. This feature can potentially be abused to read sensitive data from unencrypted internal traffic.
+author: Austin Songer
+status: experimental
+date: 2021/08/13
+references:
+    - https://cloud.google.com/kubernetes-engine/docs/how-to/audit-logging
+    - https://developers.google.com/resources/api-libraries/documentation/compute/v1/java/latest/com/google/api/services/compute/Compute.PacketMirrorings.html
+logsource:
+  service: gcp.audit
+detection:
+    selection:
+        gcp.audit.method_name: 
+            - v*.Compute.PacketMirrorings.Get
+            - v*.Compute.PacketMirrorings.Delete
```

Comment:
`Compute.PacketMirrorings.List` in the doc "call the AbstractGoogleClientRequest.execute()" like the `Compute.PacketMirrorings.Get`